### PR TITLE
Cleaning up deprecated files 

### PR DESCRIPTION
**Pull request by phiden**
See issue: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4422

### CODE REVIEW

**Review by robbiethegeek-usds — APPROVED** [review]
@annekainicUSDS can you give a quick review, once she does merge away @phiden

**Comment by ayaleloehr** [comment]
Anne is on vacation for the next week, fyi

**Comment by ayaleloehr** [comment]
@phiden Can this be merged or find a reviewer other than Anne since she is out?

**Comment by phiden** [comment]
@ayaleloehr I'll ask for another reviewer -- I know robbie wanted another set of eyes

File: content/includes/L2-section-landing.html [hunk 1]
```diff
@@ -1,26 +0,0 @@
-<div class="main" role="main">
-  <div class="section">
-    <div class="primary">
-      <div class="row">
-        <div class="small-12 columns usa-content">
-        {% if path == 'employment' %}
-          <div class="csp-inline-patch-L2-section-landing">
```
Comment (jkassemi) [inline content/includes/L2-section-landing.html]: 😄 

Can you also remove the `csp-inline-path*` rules that were added to `src/sass/_shame.scss`?

**Review by jkassemi — APPROVED** [review]
👍 - minor request in notes

**Comment by phiden** [comment]
@jkassemi rules removed, if you want to take a look!

**Pull request merged**